environment.yml

```diff
@@ -3,15 +3,16 @@ channels:
   - conda-forge
 
 dependencies:
-  # Python
   - python >=3.6
-  # Required
   - pip
   - geopandas
   - matplotlib-base
   - psycopg2
-  - postgis
   - requests
   - ipyleaflet
   - python-fastjsonschema
   - python-dotenv
+  - pip:
+    - postgis
```

Comment:
Please leave a comment why we have `postgis` in the pip section and not from conda. (Other packages here are for testing only).

Reply:
<https://pypi.org/project/postgis/> describes the PyPI `postgis` package as "PostGIS helpers for psycopg2 and asyncpg", both of which are in conda-forge under their own names. If geoDB needs them it would be better to install them from conda-forge, I think.

If we just want to avoid trying to install the unavailable postgis package on Windows, it should be enough to do

```yaml
dependencies
  - postgis  # [not win]
```

(not tested)